faq/opa.inc

```diff
@@ -0,0 +1,58 @@
+<?php
+
+include_once("$topdir/software/ompi/current/version.inc");
+
+$opa_documentation = "<a href\"https://www.intel.com/content/www/us/en/support/articles/000016242/network-and-i-o/fabric-products.html\">";
+
+$q[] = "How can the multi-rail settings be adjusted if multiple HFI (Host Fabric
+Interface) cards are installed on the system?";
+
+$anchor[] ="opa-multirail";
+
+$a[] = "
+
+Multi-Rail feature allows a process to use multiple HFIs to transfer a message
+to improve message bandwidth. PSM2 library handles the support for multi-rail
+which is off by default. The multi-rail settings can be modified using the
+following environment variables:
+
+<ul>
+
+<li> PSM2_MULTIRAIL=[0,1,2] ] (0=Disabled, 1=Enable across all HFIs in the
+system, 2=Enable multi-rail within a NUMA node) </li>
+
+<li> PSM2_MULTIRAIL_MAP=unit:port,unit:port... </li>
+
+</ul>
+
+The variables above may be included in the mpirun command line or in the
+environment. Example:
+
+<geshi bash>
+shell$ mpirun -mca mtl [psm2|ofi] -x PSM2_MULTIRAIL=1 -np 2 -H host1,host2 ./a.out
```

Comment:
may want to add a note saying that when OFI is used the PSM2 provider must be used.  Maybe also add a link to the ""How can Open MPI communicate with Intel Omni-Path Architecture (OPA)" section

Reply:
Fixed this in the new commit